internal/scan-manager/scanners/qualys/pom.xml

```diff
@@ -0,0 +1,141 @@
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.xerces</groupId>
+            <artifactId>xercesImpl</artifactId>
+            <version>${org.wso2.orbit.xerces.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>${org.apache.commons.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>${org.yaml.snakeyaml.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>${com.google.code.gson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>org.wso2.security.tools.scanmanager.common.internal</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.common.internal.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>org.wso2.security.tools.scanmanager.common</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.common.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>scanners-common</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.scanners.common.version}</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>${org.springframework.boot.maven.version}</version>
+                <configuration>
+                    <fork>true</fork>
+                    <mainClass>org.wso2.security.tools.scanmanger.scanners.qualys.App</mainClass>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>repackage</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>exec</classifier>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
```

Comment:
Shall we please add `maven-java-doc` plugin to this project with this PR. Add the commented configuration at https://github.com/wso2/security-tools/blob/master/internal/pom.xml#L264

We can incrementally add it to other projects.